Source/ACE.Server/ACE.Server.csproj

```diff
@@ -16,6 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="HttpMachine" Version="0.9.0" />
```

Comment:
Is there a way to do this with a package provided by microsoft?

Reply:
There's no .NET library to use for this outright.  I guess I'll roll my own parser for it.

Here's where I got this information:  https://stackoverflow.com/questions/975308/c-sharp-http-request-parser